node-api/src/error.rs

```diff
@@ -180,6 +241,49 @@ impl DispatchError {
 	}
 }
 
+/// An error relating to tokens when dispatching a transaction.
+#[derive(Clone, Debug, Eq, PartialEq, Encode, Decode)]
+pub enum TokenError {
```

Comment:
I guess the Error enums are taken from substrate? Can we add a link/reference to the original?